lib/cartopy/mpl/ticker.py

```diff
@@ -148,11 +257,24 @@ def __init__(self, degree_symbol=u'\u00B0', number_format='g',
             lat_formatter = LatitudeFormatter(degree_symbol='')
             ax.yaxis.set_major_formatter(lat_formatter)
 
+        When not bounded to an axis::
```

Comment:
Can you expand this comment a little please to explain what this does when not bounded to an axis?

Also I'm not sure about the use of the word 'bounded'.  The SciTools Iris package  is very often used with Cartopy, and this has a different meaning of the word 'bounded', so this may be a little confusing.  

I think it might be alright if we just use 'bound' instead, and I can't think of a better word that means the same thing.